### PR TITLE
docs: Update candidate information

### PR DESCRIPTION
**Pull request by akash1810**
Adds a note against use of GitHub co-pilot.

### CODE REVIEW

File: CANDIDATE_INFORMATION.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ Please note, enabling your camera is optional.
 
 We do not assess your IDE setup.
 
+Please note that we do not accept use of GitHub co-pilot and similar tools, as it makes it difficult for a meaningful assessment to be performed.
```
Comment (mxdvl) [inline CANDIDATE_INFORMATION.md]: I would merge the three last points into a:

```md
> **Note**
> - Enabling your camera is option
> - We do not assess your IDE setup
> - We currently forbid the use of GitHub Copilot and similar code-generation tools. They make it difficult to perform a meaningful assessment.
```

**Review by mxdvl — COMMENTED** [review]
Worth taking a position on this. We might have to review at a later date if these tools become more ubiquitous.

I would merge the last three points in a **Note**.

```md
**Note**
- Enabling your camera is optional.
- We do not assess your IDE setup.
- We currently forbid the use of Github Copilot and similar tools. They make a meaningful assessment too difficult.
```

**Review by michaelclapham — APPROVED** [review]
These changes seem sensible to me! 👍

File: CANDIDATE_INFORMATION.md [hunk 2]
```diff
@@ -21,9 +21,10 @@ Google Meet is supported by the major browsers. Full requirements can be found [
 A link will be included on your interview invitation, click it to join.
 Once joined, your interviewer will ask you to share your screen.
 
-Please note, enabling your camera is optional.
-
-We do not assess your IDE setup.
+> **Note**
+> - Enabling your camera is optional.
+> - We do not assess your IDE setup.
+> - We currently forbid the use of GitHub Copilot and similar tools. They make it difficult to perform a meaningful assessment.
```
Comment (ab-gnm) [inline CANDIDATE_INFORMATION.md:27]: Suggest removing the reasoning for forbidding Github Copilot. Most candidates would understand it, or ask about it. We could provide this statement as an guideline explanation in the [PROCESS_REMOTE.md](https://github.com/guardian/coding-exercises/blob/main/PROCESS_REMOTE.md)

```suggestion
> - Turning on your camera is optional.
> - We do not assess your IDE setup.
> - We currently do not allow the use of GitHub Copilot and similar tools.
```

**Review by ab-gnm — APPROVED** [review]

**Pull request merged**